src/analysis/retail/paladin/holy/modules/talents/GlimmerOfLight/GlimmerOfLight.tsx

```diff
@@ -33,31 +33,51 @@ import BeaconHealSource from '../beacons/BeaconHealSource';
  */
 
 const BUFF_DURATION = 30;
-const GLIMMER_CAP = 8;
 
 class GlimmerOfLight extends Analyzer {
   static dependencies = {
     beaconHealSource: BeaconHealSource,
   };
 
+  GLIMMER_CAP = 3;
+
   casts = 0;
-  damage = 0;
+  damageCast = 0;
```

Comment:
new name seems like a cast counter but actually it's a damage accumulator

Reply:
errrr yeah it kind of does...
I'll think of something... maybe `damageHS` HS short of holy shock